static/app/views/settings/organizationAuditLog/index.tsx

```diff
@@ -105,15 +106,19 @@ function OrganizationAuditLog({location}: Props) {
 
   return (
     <Fragment>
-      <AuditLogList
-        entries={state.entryList}
-        pageLinks={state.entryListPageLinks}
-        eventType={state.eventType}
-        eventTypes={state.eventTypes}
-        onEventSelect={handleEventSelect}
-        isLoading={state.isLoading}
-        onCursor={handleCursor}
-      />
+      {!organization.access.includes('org:write') ? (
+        <PermissionAlert />
```

Comment:
nit: should we pass in `org:write` here? the default right now is `project:write`. the message is correct though